content/etc/icingaweb2/config.ini

```diff
@@ -0,0 +1,10 @@
+[global]
+show_stacktraces = "1"
+config_backend = "db"
+config_resource = "icingaweb_db"
+module_path = "/usr/share/icingaweb2/modules:/etc/icingaweb2/modules"
```

Comment:
There is the `module_path` for icingaweb2. It just works like a normal path-variable and tells icignacli/icingaweb2 where to search for plugins. I think we have to define 3 locations:
- for self-installed modules from the user -> `/etc/icingaweb2/modules-user` (on mounted volume)
- for the ones installed from docker-container -> `/opt/icingaweb2/modules` (not on volume) 
- for the ones by debian-packages -> `/usr/share/icingaweb2/modules`